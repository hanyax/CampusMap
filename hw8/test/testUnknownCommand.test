# Test unknown command and menu
r r
abc
m

# All of your test scripts should end with q to terminate the program.
q
