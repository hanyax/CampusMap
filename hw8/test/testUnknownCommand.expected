Menu:
r to find a route
b to see a list of all buildings
q to quit

Enter an option ('m' to see the menu): # Test unknown command and menu
Unknown option

Enter an option ('m' to see the menu): Unknown option

Enter an option ('m' to see the menu): Menu:
r to find a route
b to see a list of all buildings
q to quit

Enter an option ('m' to see the menu):
# All of your test scripts should end with q to terminate the program.
